lib/mvi/responses/base.rb

```diff
@@ -43,6 +44,11 @@ def body
 
       private
 
+      def ensure_xml_prolog(xml)
+        xml.prepend('<?xml version="1.0" encoding="UTF-8"?>') unless xml =~ /^<\?xml/
```

Comment:
1. The response that did include a prolog had the encoding set to UTF-8 so ya was assuming that was the case in the other responses.
2. Yes response.body is always XML